doc/userguide/rules/base64-keywords.rst

```diff
@@ -0,0 +1,53 @@
+Base64 keywords
+===============
+
+Suricata supports decoding base64 encoded data from buffers and matching on the decoded data.
+
+This is achieved by using two keywords, ``base64_decode`` and ``base64_data``. Both keywords must be used in order to generate an alert.
+
+These keywords are compatible with their Snort counterparts.
```

Comment:
while it's not bad to mention that, we also have a page for differences. Not sure if it's necessary and if it is, it might be added to other keywords as well.

Reply:
I agree. It's better to be consistent.